coordinator/src/node.rs

```diff
@@ -395,15 +365,13 @@ impl Node {
                         );
 
                         let channel = self.inner.get_dlc_channel_by_id(&channel_id)?;
-                        let contract_id =
-                            channel.get_contract_id().context("missing contract id")?;
 
                         let protocol_executor =
                             dlc_protocol::DlcProtocolExecutor::new(self.pool.clone());
                         protocol_executor.finish_dlc_protocol(
                             protocol_id,
                             &node_id,
-                            &contract_id,
+                            channel.get_contract_id(),
```

Comment:
:wrench: Maybe we could just add a comment (and a named variable) for all these.

Reply:
Not sure what you mean. Do you want a separate variable for the `contract_id`?